shamir_mnemonic/__init__.py

```diff
@@ -271,7 +283,7 @@ def _encrypt(master_secret, passphrase, iteration_exponent, identifier):
     return r + l
 
 
-def _decrypt(encrypted_master_secret, passphrase, iteration_exponent, identifier):
+def decrypt(identifier, iteration_exponent, encrypted_master_secret, passphrase):
```

Comment:
I would prefer to keep the parameter order the way it was, simply because that's the order which appears in the spec. I am not sure why the order is different in the core, probably my bad.